test/description-render-test.js

```diff
@@ -266,11 +269,71 @@ describe('Markdown', function() {
         var expectedHtml, markdownString;
         markdownString = '<img src="/image.jpg" onerror="alert(\'XSS\')" />';
         expectedHtml = '<img src="/image.jpg">\n';
-        markdownRenderer.renderHtml(markdownString, function(error, html) {
+        markdownRenderer.renderHtml(markdownString, { sanitize: sanitize }, function(error, html) {
           assert.strictEqual(html, expectedHtml);
           done(error);
         });
       });
+      context('given <a> tag', function () {
```

Comment:
Please add a new line before this:
```suggestion

      context('given <a> tag', function () {
```

Reply:
changed